omod/src/main/resources/apps/createUser_extension.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "id": "${project.parent.groupId}.${project.parent.artifactId}.createUser.customFragments.userLocation",
+    "extensionPointId": "createUser.manageAccountUserFragments",
+    "extensionParams": {
+      "title" : "Location",
+      "type": "userProperty",
+      "userPropertyName" : "locationId",
+      "label": "Select Location",
```

Comment:
Shouldn't we localise these strings?

Reply:
Yes, we need to do it. Since other extensions also need to have it, Can we do these all with a different ticket?

Reply:
I thought you would just replace them with message codes hence not requiring a separate ticket.